ChangeLog

```diff
@@ -1,3 +1,7 @@
+Version 1.18.1
+--------------
+    * Disallow empty files being added in `cgo_library`
```

Comment:
Nit: We're not really disallowing anything here. Perhaps something like:

```suggestion
    * Filter out empty output lines in `cgo_library`
```

Ditto PR title